### PR TITLE
working-groups: update table of facilitators

### PR DESCRIPTION
**Pull request by vdemeester**
Adding facilitators and removing some others so that we are in sync
with the present.

/cc @bobcatfish @pritidesai @nikhil-thomas @savitaashture @khrm @afrittoli @piyush-garg @pradeepitm12 

Signed-off-by: Vincent Demeester <vdemeest@redhat.com>

### CODE REVIEW

File: working-groups.md [hunk 1]
```diff
@@ -122,9 +122,10 @@ This is the [`tektoncd/cli`](https://github.com/tektoncd/cli) working group.
 
 | &nbsp;                                                        | Facilitators      | Company | Profile                                           |
 | --------------------------------------------------------      | ----------        | ------- | ---------------------------------------           |
-| <img width="30px" src="https://github.com/vdemeester.png">    | Vincent Demeester | Red Hat | [vdemeester](https://github.com/vdemeester)       |
 | <img width="30px" src="https://github.com/chmouel.png">       | Chmouel Boudjnah  | Red Hat | [chmouel](https://github.com/chmouel)             |
 | <img width="30px" src="https://github.com/danielhelfand.png"> | Daniel Helfand    | VMware  | [danielhelfand](https://github.com/danielhelfand) |
+| <img width="30px" src="https://github.com/pradeepitm12.png"   | Pradeep Kumar     | Red Hat | [pradeepitm12](https://github.com/pradeepitm12)   |
```
Comment (bobcatfish) [inline working-groups.md:127]: i think there's a slight error here:

image

File: working-groups.md [hunk 2]
```diff
@@ -142,14 +143,12 @@ repository that need to expose documentation.
...
-| <img width="30px" src="https://github.com/spomorski.png">  | Serge Pomorski    | Google  | [spomorski](https://github.com/spomorski)   |
-| <img width="30px" src="https://github.com/vdemeester.png"> | Vincent Demeester | Red Hat | [vdemeester](https://github.com/vdemeester) |
-| <img width="30px" src="https://github.com/anneqm.png"> | Anne Michalak | IBM | [anneqm](https://github.com/anneqm) |
-| <img width="30px" src="https://github.com/popcor255.png"> | Napoleon Santana | IBM | [popcor255](https://github.com/popcor255) |
-| <img width="30px" src="https://github.com/AlanGreene.png">    | Alan Greene     | IBM     | [AlanGreene](https://github.com/AlanGreene)
+| &nbsp;                                                     | Facilitators     | Company | Profile                                     |
+| --------------------------------------------------------   | ----------       | ------- | ---------------------------------------     |
+| <img width="30px" src="https://github.com/spomorski.png">  | Serge Pomorski   | Google  | [spomorski](https://github.com/spomorski)   |
```
Comment (bobcatfish) [inline working-groups.md:148]: looks like this was already broken (probably i did this XD), https://github.com/spomorski isnt right, i think this should be https://github.com/sergetron

File: working-groups.md (hunk 1 above)
Comment (vdemeester) [reply working-groups.md:127]: ah yeah it's missing a `>`..

File: working-groups.md (hunk 2 above)
Comment (vdemeester) [reply working-groups.md:148]: :+1:

**Comment by pritidesai** [comment]
Very exciting 🎉 
/lgtm

**Comment by dibyom** [comment]
/hold cancel

(The avatar looks ok now!)

**Pull request merged**